src/app/components/Flow/Flow.tsx

```diff
@@ -37,13 +40,14 @@ export const Flow = ({ components, onDone }: FlowProps) => {
 
   return (
     <>
-      <FlowHeader
-        onPrevious={handlePrevious}
-        steps={components.length}
-        currentStep={step}
-        mt="3"
-        mb="2"
-      />
+      <Container pt="4" sx={{ flex: 'none' }}>
```

Comment:
I think there is too much padding here too, even though the designs look similar.